feature_extraction/chi_squared.ipynb

```diff
@@ -0,0 +1,863 @@
+{
```

Comment:
<p>NaNs are pretty sus here...</p><p> </p><p>We should investigate and address this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/treyduffus/multiclass_lungcancer_classification_models/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
fixed the NaN issue in chi-squared analysis and added better validation checks